tests/e2e/ctl_v3_grpc_test.go

```diff
@@ -127,20 +127,20 @@ func TestAuthority(t *testing.T) {
 					t.Fatalf("could not start etcd process cluster (%v)", err)
 				}
 				defer epc.Close()
-				endpoints := templateEndpoints(t, tc.clientURLPattern, epc)
 
-				client, err := e2e.NewEtcdctl(cfg.Client, endpoints)
-				assert.NoError(t, err)
-				err = client.Put(ctx, "foo", "bar", config.PutOptions{})
-				if err != nil {
-					t.Fatal(err)
+				endpoints := templateEndpoints(t, tc.clientURLPattern, epc)
+				client := newClient(t, endpoints, cfg.Client)
```

Comment:
Why change this? e2e test is supposed to use etcdctl to talk to etcdserver.

Reply:
The possibility of using `etcdctl` to round-robin requests to different etcd servers is low. 
```
etcdctl --endpoints IP_1, IP_2, IP3 put key foo
etcdctl --endpoints IP_1, IP_2, IP3 put key foo
etcdctl --endpoints IP_1, IP_2, IP3 put key foo
```
is likely send all the requests to IP_1. 

It is because https://github.com/etcd-io/etcd/blob/c6fd719ccf2891b9e8c61dbb6bbc9ba63abbdd12/tests/framework/e2e/etcdctl.go#L154-L162 establish a short-lived connection and gRPC client usually picks the first IP to connect with. 

So I would like to use `clientv3.Client` to establish a long-lived connection and ensure client requests are evenly sent to multiple servers.

Reply:
This test was explicitly added to test `etcdctl` and not `clientv3`. There is matching integration tests for `clientv3`. Please keep the `etcdctl` here.

Reply:
> There is matching integration tests for `clientv3`

okay. It's fair. I will try workaround based on `etcdctl`.